test/__tests__/data/marketo_router_output.json

```diff
@@ -345,38 +353,38 @@
         "clientSecret": "3l3gJpzRsZagD6gu7tnTeKXz0bomLGnd",
         "trackAnonymousEvents": false,
         "createIfNotExist": true,
-        "customActivityEventMap": [
+        "rudderEventsMapping": [
           {
-            "from": "acq_signup_completed",
-            "to": "100026"
+            "event": "acq_signup_completed",
+            "marketoActivityId": "100026"
           },
           {
-            "from": "act_createwebinarform_submit",
-            "to": "100025"
+            "event": "act_createwebinarform_submit",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_presentation_style",
-            "to": "100025"
+            "event": "act_presentation_style",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_webinar_view",
-            "to": "100025"
+            "event": "act_webinar_view",
+            "marketoActivityId": "100025"
```

Comment:
Why is `marketoPrimaryKey` not present in these cases ?

Reply:
We have manipulated these using a script previously as well it won't be present and thats why it is not present currently as well